templates/css/example.css

```diff
@@ -96,7 +96,6 @@ code {
 }
 .abe-preview {
   background: #fff;
-  overflow: hidden;
```

Comment:
why did you remove this?

Reply:
Because otherwise the calendar was not showing as an overlay.

Reply:
Does this affect the way previews are rendered in other samples?

Reply:
I haven't found any problem yet